README.md

```diff
@@ -207,7 +201,7 @@ Example:
 
 👤 **Author1**
 
-- GitHub: [@githubhandle](https://github.com/githubhandle)
+- GitHub: [@githubhandle](https://github.com/lawrahkonwea)
 - Twitter: [@twitterhandle](https://twitter.com/twitterhandle)
 - LinkedIn: [LinkedIn](https://linkedin.com/in/linkedinhandle)
 
```

Comment:
Kindly ensure to improve upon the following aspects of your ReadMe (which are required review criteria), like informed in the previous review:
- [ ] OPTIONAL
Kindly ensure to remove the template logo. Optionally, you can replace it with your own logo if you have one.
- [ ] OPTIONAL
Kindly ensure that the _Future Features_ section of your ReadMe includes 1-3 potential features to be added later.
- [ ] OPTIONAL
This project appears to be worked on by a sole contributor, therefore kindly update the Author section of the ReadMe, remove redundant contributor(s) details and change the header title _Authors_ to _Author_ accordingly. 
- [ ] OPTIONAL
Kindly endeavor to fill the FAQ section with at least 2 Q&A(s).
- [ ] OPTIONAL
 MIT license link in your ReadMe is broken, kindly endeavor to fix it, and remove the guide note below it.

Reply:
Thank you